acceptance/suites/pre_suite/foss/30_install_dev_repos.rb

```diff
@@ -26,11 +26,12 @@
   end
 end
 
-if puppetdb_supported_platforms.include?(master.platform) then
-  step "Install PuppetDB repository" do
-    install_puppetlabs_dev_repo(
-      master, 'puppetdb', test_config[:puppetdb_build_version],
-      repo_config_dir, install_opts)
+puppetdb_supported_platforms.each do |pdb_platform|
+  if master.platform.include?pdb_platform.source then
```

Comment:
`if master.platform ~= pdb_platform` syntax errors.  

When master.platform = "el-6-x86_64" and pdb_platform.source = "el" , 
`master.platform =~ pdb_platform` returns zero (0).
`master.platform =~ pdb_platform.source` returns type mismatch